server/src/main/java/org/candlepin/async/tasks/HypervisorUpdateJob.java

```diff
@@ -23,7 +23,7 @@
 import org.candlepin.async.JobExecutionContext;
 import org.candlepin.async.JobExecutionException;
 import org.candlepin.auth.Principal;
-import org.candlepin.common.exceptions.BadRequestException;
+import org.candlepin.common.resource.exceptions.BadRequestException;
```

Comment:
Jobs and other non-resource parts of code shouldn't be throwing resource exceptions, but common ones. I assume this is a TODO right?

Reply:
Exactly. It should be done as we go over the resources, extracting the controllers.